cuda_core/cuda/core/experimental/_memory/_device_memory_resource.pyx

```diff
@@ -49,8 +48,8 @@ cdef class DeviceMemoryResourceOptions:
         Maximum pool size. When set to 0, defaults to a system-dependent value.
         (Default to 0)
     """
-    ipc_enabled : cython.bint = False
-    max_size : cython.size_t = 0
+    ipc_enabled : bool = False
+    max_size : int = 0
```

Comment:
I took the opportunity to also fix the signature errors (we don't want users to see Cython here):
https://nvidia.github.io/cuda-python/cuda-core/0.4.2/generated/cuda.core.experimental.DeviceMemoryResourceOptions.html